packages/nimbus/src/components/tooltip/tooltip-dev.mdx

```diff
@@ -0,0 +1,188 @@
+| `title` prop | `<Tooltip.Content>` children | Text content now goes as children |
+| `isOpen` prop | `isOpen` prop | ✅ Same API |
+| `onClose` | `onOpenChange` | ⚠️ Renamed, now receives boolean parameter |
+| `horizontalConstraint` | Not supported | ❌ Removed - use CSS or container sizing |
+
+
+#### Deprecated props
+- `onClose` → `onOpenChange`: Now receives `(isOpen: boolean) => void` instead of `() => void`
+- `title` prop → Content as children: Text now goes inside `<Tooltip.Content>`
+- `delay` and `closeDelay`: More granular control over timing
+- `horizontalConstraint`: Use CSS width constraints on content instead
+- `tone`: Styling is now handled through design tokens
+
+## Testing Strategies and Best Practices
+
+#### 1. Unit Testing with React Testing Library
+
+**Recommended Testing Library Setup:**
+```tsx
+import { render, screen, userEvent } from '@testing-library/react';
+import { NimbusProvider } from '@commercetools/nimbus';
+import { IntlProvider } from 'react-intl';
+import { Tooltip, Button } from '@commercetools/nimbus';
+
+// Test wrapper with required providers
+const TestWrapper = ({ children }: { children: React.ReactNode }) => (
+  <IntlProvider locale="en" messages={{}} defaultLocale="en">
+    <NimbusProvider>
+      {children}
+    </NimbusProvider>
+  </IntlProvider>
+);
+
+// Custom render function
+const renderWithProviders = (ui: React.ReactElement) => 
+  render(ui, { wrapper: TestWrapper });
+```
+
+#### 2. Storybook Integration for Testing
+
+Use the [Nimbus storybook Tooltip component story](https://nimbus-storybook-6l4m4g78g-commercetools.vercel.app/?path=/docs/components-tooltip-tooltip--docs) as a playground for visual testing of the component. 
+
+#### 3. Regression tests
+
+### 4. Accessibility Standards
+
+Nimbus Tooltip automatically handles most accessibility requirements, but developers should be aware of these standards:
+
+#### Implementation Notes
+- Screen reader compatible with proper ARIA attributes
+- Keyboard navigation support built-in
+
+## Quick Reference Summary
+
+### Essential Patterns
+- **Compound Components**: Always use `<Tooltip.Root>` + `<Tooltip.Content>`
+- **Testing**: Include both providers in test wrapper
+- **Accessibility**: Built-in ARIA support, keyboard navigation included
+- **Performance**: Memoize handlers and styles for optimal performance
+
+### Common Code Patterns
+
+#### Hooks
+```tsx
+// Custom Hook Pattern
+const useTooltip = (content: string, options = {}) => { ...};
+
+// Using the Custom Hook
+const MyComponent = () => {
+  const { tooltipProps, contentProps } = useTooltip(
+    "This action will save your changes",
+    { placement: "bottom", delay: 300 }
+  );
+
+  return (
+    <Tooltip.Root {...tooltipProps}>
+      <Button>Save</Button>
+      <Tooltip.Content {...contentProps} />
+    </Tooltip.Root>
+  );
+};
+```
+
+
+### API Reference
+
+<PropsTable id="Tooltip" />
```

Comment:
We still need a different topic to discuss how we would manually add props. The exposed props listing on the props table does not capture a lot of the important props.